src/css/components/Button.scss

```diff
@@ -38,243 +36,96 @@ $Button-padding: 0 scaleGrid(2);
   height: $Button-height;
   line-height: $Button-line-height;
   margin: 0;
+  min-width: $Button-height;
   padding: $Button-padding;
   text-decoration: none;
   transition: all $duration-short ease-in-out;
   vertical-align: middle;
   white-space: nowrap;
 
   &:hover,
-  &:active,
   &:focus {
-    background-color: $Button-bg-hover;
-    border-color: $Button-border-color-hover;
-    color: $Button-color;
+    border-color: blue(5);
+    color: blue(5);
     outline: none;
   }
 
-  &.is-active {
-    background-color: $Button-bg-hover;
-    border-bottom-width: 1px;
-    border-top-width: $Button-border-bottom-width;
-    color: $Button-color;
+  &:active, &.is-active {
+    border-color: blue(7);
+    color: blue(7);
   }
 
   &.is-disabled,
   &:disabled {
     &,
     &:hover,
     &:active {
-      background-color: $Button-bg;
-      border-color: $Button-border-color;
+      background-color: gray(2);
+      border-color: gray(5);
       cursor: not-allowed;
-      color: $Button-color;
-      opacity: $Button-disabled-opacity;
+      color: gray(5);
     }
   }
 }
 
+$permutations: (
+  "primary" blue(6) blue(5) blue(7),
+  "blue" blue(6) blue(5) blue(7),
+  "negative" red(6) red(5) red(7),
+  "orange" orange(6) orange(5) orange(7),
+  "green" green(6) green(5) green(7)
```

Comment:
Not a big deal, but just thinking about consistency of naming. Do we want "green" to be positive if red is "negative"? The green button may have more use cases outside of the positive/negative context, so I could see that as an argument for keeping it "green".

Reply:
These match the existing button classes that we had for parity reasons. I am okay with changing these strictly to primary, negative, warning, and positive. I think that would technically be more clear with theming too.

Reply:
May want to loop @l3aconator and @andrewmeacham into this convo. Do we expect that we will use orange and green buttons outside of the context of positive/warning?

Reply:
Green will be used to indicate "positive" states and actions, but also may be used as a secondary accent color for things like charts and graphs.

Reply:
Orange will be used to surface warnings, but may also be used for things like badging and alerting the user to take an action.

Reply:
What about something like `--primary` (blue), `--secondary` (green),  `--tertiary` (orange), `--negative` (red) to accommodate broader use cases and future theming?

Reply:
Sorry for being wishy washy! But I'm on the fence on this one. I do still like primary, negative, warning and positive since those names have meaning associated with them and would promote consistency in theming and usage.

Reply:
That works for me. I don't have a strong opinion on this, but your proposal makes sense for theming -- which I think we will want to expand on in future cycles.